shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -1511,11 +1511,90 @@ - (void)setEditableTransform:(NSArray*)matrix {
   _cachedFirstRect = kInvalidFirstRect;
 }
 
+- (BOOL)transformPoint:(CGPoint)point
+              isAffine:(BOOL)isAffine
+                  minX:(CGFloat&)minX
+                  maxX:(CGFloat&)maxX
+                  minY:(CGFloat&)minY
+                  maxY:(CGFloat&)maxY {
+  CGFloat x =
+      _editableTransform.m11 * point.x + _editableTransform.m21 * point.y + _editableTransform.m41;
+  CGFloat y =
+      _editableTransform.m12 * point.x + _editableTransform.m22 * point.y + _editableTransform.m42;
+  if (!isAffine) {
+    const CGFloat w = _editableTransform.m14 * point.x + _editableTransform.m24 * point.y +
+                      _editableTransform.m44;
+    if (w == 0.0) {
+      return NO;
+    }
+    x /= w;
+    y /= w;
+  }
+  minX = MIN(minX, x);
+  maxX = MAX(maxX, x);
+  minY = MIN(minY, y);
+  maxY = MAX(maxY, y);
+  return YES;
+}
+
+// Returns the bounding CGRect of the transformed incomingRect, in the view's
+// coordinates.
+- (CGRect)localRectFromFrameworkTransform:(CGRect)incomingRect {
+  CGFloat minX = CGFLOAT_MAX;
+  CGFloat minY = CGFLOAT_MAX;
+  CGFloat maxX = -CGFLOAT_MAX;
+  CGFloat maxY = -CGFLOAT_MAX;
```

Comment:
If we update `transformPoint` to use `CGRect` then it might be possible to use:
```objc
CGRect maxRect = CGRectInfinite;
```

You'd want to verify that all the other logic is correct with respect to infinite values though.